ds_judgements_public_ui/templates/pages/about_this_service.html

```diff
@@ -130,248 +139,322 @@ <h2 id="section-justice">Open justice</h2>
-    corrections to the text of currently published judgments can only be made by the court by sending a replacement
-    version of the judgment through the TDR service.
-  </p>
-  <h3>Take down requests</h3>
-  <p>
-    The National Archives take down and reclosure policy does not apply to judgments or decisions published on the Find
-    Case Law Service.
-  </p>
-  <p>
-    Once a judgment or decision has been published, we will only withdraw it from publication (or replace it with a
-    revised version) if we are instructed to do so by the court or tribunal.
-  </p>
-  <p>
-    If you believe that a judgment or decision should not be publicly available to you should contact the court or
-    tribunal directly.
-  </p>
-  <p>
-    Please refer to our full publishing policy for more details <a href='{% url "publishing_policy" %}'>{% translate "publishing_policy.title" %}</a>.
-  </p>
-  <h2 id="section-contact">How to contact us</h2>
-  <p>
-    You can get in touch by emailing
-    <a href="mailto:{% translate "caselaw.email" %}">{% translate "caselaw.email" %}</a>.
-  </p>
-{% endblock static_content %}
+    When we publish a judgment or decision we automatically identify and hyperlink citations to other
+    judgments we have or to legislation. We refer to this process as enrichment.
+  </p>
+  <p>
+    Links to legislation will take you to the legislation as it stands today and you will need to use their
+    ‘point in time’ feature to find the legislation as it stood at a particular date.
+  </p>
+  <h2 id="section-data">How to access the data and technical documentation</h2>
+  <p>Licence terms apply to all data re-use. You can read more about our licences</p>
+  <p>
+    To retrieve the LegalDocML data for judgments, you can use the API. The Data is accessible through
+    Atom feeds allowing you to keep track of recently published or updated judgments. You can find
+    more details and guidelines in the OpenAPI specification, which available on the.  
+    <a href="https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/openapi/public_api.yml">Case Law service
+      GitHub Project page.</a>.
+    </p>
+    <p>Example root feed URL:</p>
+    <p>
+      <a href="https://caselaw.nationalarchives.gov.uk/atom.xml">https://caselaw.nationalarchives.gov.uk/atom.xml</a>.
+    </p>
+    <p>URL Structure for Judgment Data:</p>
+    <p>
+      <a href="https://caselaw.nationalarchives.gov.uk/year/court/subdivision/id/data.xml">https://caselaw.nationalarchives.gov.uk/year/court/subdivision/id/data.xml</a>.
+    </p>
+    <p>In most cases the Neutral citation is how we uniquely identify the judgments and decisions.</p>
+    <h3>Parameters</h3>
+    <ul>
+      <li>
+        <code>order = updated :</code>  Sort by the last XML database update
+      </li>
+      <li>
+        <code>order = date :</code> daSort by date on the judgment
```

Comment:
as above

Reply:
![Screenshot from 2024-01-12 15-00-37](https://github.com/nationalarchives/ds-caselaw-public-ui/assets/75584408/bea754bf-14a5-4278-8f5c-c08cff5969d0)
Thank you updated @dragon-dxw